Gemfile

```diff
@@ -76,6 +76,7 @@ gem 'mediaelement-title', git:'https://github.com/avalonmediasystem/mediaelement
 gem 'mediaelement-track-scrubber', git: 'https://github.com/avalonmediasystem/mediaelement-track-scrubber.git', tag: 'avalon-r6'
 
 # Jobs
+gem 'redis', '~> 3.3.5'
```

Comment:
This is new?

Reply:
The redis  >=4.0 gem causes an error 'NoMethodError: undefined method for Redis:Class'

My though, pin the redis gem and add a technical debt issue? Thoughts?

Reply:
> It looks like all the gems were updated in Gemfile.lock. Was that intentional? It would solve all the outstanding PRs created by dependabot including several other Security bumps (sprockets, loofah, nokogiri, bootstrap-sass, ffi, devise and rack)
> 

Yes, intentional. 

> I just noticed this is WIP, sorry for the early review. Let me know if you want a second set of eyes for debugging the travis issue.

Thank you for the early review. I added a WIP PR to make the life easier when asking questions and getting feedback.